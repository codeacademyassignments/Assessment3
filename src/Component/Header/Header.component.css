.header{
  display: flex;
  flex-direction: rowl;
  justify-content: space-between;
}
.body{
  background-color: grey;
  height: 100%;
}
.firstDiv,.thirdDiv{
  align-self: center;
  width:45%;
  color: white;
}

.secondDiv{
  align-self: center;
}


