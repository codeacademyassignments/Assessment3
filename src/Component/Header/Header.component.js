import './Header.component.css';
import React from 'react';

const Header = () => (
  <header className="header">
    <div className="firstDiv"><hr /></div>
    <div className="secondDiv">

      <svg width="100" height="100">

        <circle cx="50" cy="50" r="30" stroke="white" strokeWidth="3" fill="grey" />
        <text x="50%" y="50%" textAnchor="middle" stroke="white" strokeWidth="1px" dy=".3em">BS</text>
      </svg>
    </div>
    <div className="thirdDiv"><hr /></div>
  </header>
);

export default Header;
