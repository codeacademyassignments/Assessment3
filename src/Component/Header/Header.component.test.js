import renderer from 'react-test-renderer';
import React from 'react';
import Header from './Header.component';

describe('Header:', () => {
  it('should render previous snapshot', () => {
    const tree = renderer.create(<Header />);
    expect(tree).toMatchSnapshot();
  });
});
