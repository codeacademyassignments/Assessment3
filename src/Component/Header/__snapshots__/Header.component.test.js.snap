// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header: should render previous snapshot 1`] = `
<header
  className="header"
>
  <div
    className="firstDiv"
  >
    <hr />
  </div>
  <div
    className="secondDiv"
  >
    <svg
      height="100"
      width="100"
    >
      <circle
        cx="50"
        cy="50"
        fill="grey"
        r="30"
        stroke="white"
        strokeWidth="3"
      />
      <text
        dy=".3em"
        stroke="white"
        strokeWidth="1px"
        textAnchor="middle"
        x="50%"
        y="50%"
      >
        BS
      </text>
    </svg>
  </div>
  <div
    className="thirdDiv"
  >
    <hr />
  </div>
</header>
`;
